ml/base/config.py

```diff
@@ -38,6 +39,27 @@ def from_dict(cls, dic):
     def from_internal_file(cls, config_name):
         pass
 
+    @classmethod
```

Comment:
In general it's bad form to write pseudo-code like real code as it can be confusing for people to tell them apart. I usually write it with comments, then later you can add code between the comments and then delete / modify the comments as you see fit:
```python
def some_real_api():
    # while something is true:
         # I'll do this and that
    pass
```

Reply:
You are doing operations on the object itself so it's just normal method, not classmethod